compiler/rustc_parse/src/parser/diagnostics.rs

```diff
@@ -573,6 +574,14 @@ impl<'a> Parser<'a> {
             return Err(self.sess.create_err(UseEqInstead { span: self.token.span }));
         }
 
+        if self.token.ident().map(|(ident, _)| ident.name == kw::Move).unwrap_or_default()
```

Comment:
Can you use `is_some_and` here and below?

Reply:
Also, you can use https://doc.rust-lang.org/nightly/nightly-rustc/rustc_ast/token/struct.Token.html#method.is_keyword here (at least for `move`)

Reply:
Thanks